tests/sentry/backup/test_correctness.py

```diff
@@ -0,0 +1,188 @@
+import io
+import os
+from pathlib import Path
+from typing import Dict, List, NewType, Optional, Protocol, Tuple
```

Comment:
if you `from __future__ import annotations` you can use `dict[K, V]` / `list` / `| None` / `tuple` instead of `Dict`, `List`, `Optional`, `Tuple`

Reply:
Done.